add warning/note for use of parenetheses (#8805)
DataDog/documentation: content/en/monitors/notifications.md

| [Slack][7]     | `@slack`     | [Examples][8]  |
| [Webhooks][9]  | `@webhook`   | [Examples][10] |

See the [list of integrations][11] that can be used to notify your team.

> **Note**: Currently handles which include parentheses (`(`, `)`) are not supported. When a handle with parentheses is used the handle will not be parsed and no alert will be created.

Reviewer: ```suggestion
**Note**: Handles that include parentheses (`(`, `)`) are not supported. When a handle with parentheses is used, the handle is not be parsed and no alert is created.
```
